src/elements/ScrollContainer/ScrollContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import { SpacerInline } from '@elements/SpacerInline';
+import EndOfScrollWrapper from './EndOfScrollWrapper';
+import Checkbox from '../Checkbox/Checkbox';
+import styles from './ScrollContainer.styles';
+
+interface ScrollContainerProps {
+	onScrollToEnd?: Function;
+	onPress?: Function;
+	setChecked: Function;
+	documentText: string;
+	checked: boolean;
+	checkboxSize?: number;
+	displayCheckbox?: boolean;
+	checkboxLabel?: string;
+}
+
+export default function ScrollContainer({
+	onScrollToEnd = () => {},
+	onPress = () => {},
+	setChecked,
+	checkboxSize,
+	documentText,
+	displayCheckbox = true,
+	checked,
+	checkboxLabel,
+}: ScrollContainerProps) {
+	const handleDisplayCheckbox = () => {
+		if (displayCheckbox) {
+			return (
+				<View style={styles.checkboxContainer}>
+					<Checkbox
```

Comment:
For some reason when I test your branch the `Icon` element that's supposed to show a `CheckboxOff/CheckboxOn` is showing a `Camera` icon.
Do you see the same thing when you test?

Either way, based on [prealpha figma design](https://www.figma.com/file/Kzv8b6CHHq5Y5aCuBWAGLU/THE-BANANA-APP-BGP?node-id=5374%3A36) I don't think we need this `Checkbox` anymore right?

Reply:
Hah, you're right! I didn't see that they had updated the figma design. I'll change the component to match the new design.

And I was getting the same camera icon, but I didn't fix it because I believe Heather was working on a PR to fix that.

Reply:
Oh gotcha!